File: mixin-base-test/src/test/java/net/questcraft/apitests/MixinTestCase.java
package net.questcraft.apitests;


import net.yakclient.mixin.api.Injection;
import net.yakclient.mixin.api.Mixer;
import org.objectweb.asm.Opcodes;

@Mixer("net.questcraft.apitests.MixinSourceClassTest")
public class MixinTestCase {
//    private int shadowInt;
//    private String somethingElse;

//    @Shadow
//    public abstract void shadowMethod();
//
//    public abstract void second();
//
//    public static void staticTest() {
//        System.out.println("The current class is this");
//    }

    @Injection(type = Opcodes.GETSTATIC)
    public void printTheString() {
        final var otherString = "clashing?";
       System.out.println(otherString);
    }
}

File: mixin-base-test/src/test/kotlin/net/questcraft/apitests/MixinTestCase.kt
package net.questcraft.apitests

import net.yakclient.mixin.api.Injection
import net.yakclient.mixin.api.Mixer

@Mixer("net.questcraft.apitests.MixinSourceClassTest")
class MixinTestCase {
    //    private int shadowInt;
    //    private String somethingElse;
    //    @Shadow
    //    public abstract void shadowMethod();
    //
    //    public abstract void second();
    //
    //    public static void staticTest() {
    //        System.out.println("The current class is this");
    //    }
    @Injection
    fun printTheString() {
        val otherString = "clashing?"
        println(otherString)
    }
}